scripts/sql/apolloconfigdb.sql

```diff
@@ -446,3 +445,45 @@ VALUES
 /*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
 /*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
 /*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
+
+
+DROP TABLE IF EXISTS `AuditLog`;
+
+CREATE TABLE `AuditLog` (
```

Comment:
@apolloconfig/committers For backward compability as more as possible

The solution is that

* create new table, the new audit log only save in the new table.
* open this feature by config `apollo.audit.log.enabled=true`, and user can close it by change to `apollo.audit.log.enabled=false`
* create new maven module apollo-audit, other modules use this feature by annotation or interface `ApolloAuditLogApi `.

Is it ok?